installation/docker-commands/create-gateway.sh

```diff
@@ -26,11 +26,15 @@ fi
 
 # Ask the user for the hummingobt data folder location
 prompt_hummingbot_data_path () {
-read -p "   Enter the full location path where your Hummingbot files are located (example: /Users/hbot/hummingbot_files) >>> " FOLDER
+read -p "   Enter the location where your Hummingbot files are located (example: /Users/hbot/hummingbot_files) >>> " FOLDER
 if [ "$FOLDER" == "" ]
 then
   prompt_hummingbot_data_path
 else
+  if ! [[ "$FOLDER" == *"$HOME"* ]]; then
```

Comment:
The correct method to detect whether a path is absolute or relative is to look for a leading "/". If there's a leading "/", then it's absolute. Otherwise, it's relative.

Reply:
Another thing is, since this detection and conversion function is common across a few scripts - it'll be good to refactor it to a bash function in a common utilities library file - and source that file in all the scripts that needs it.